framework/test/unit/components/select/get-matching-value-test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai'
+import { getMatchingValue } from '../../../../lib/components' // adjust the path accordingly
+
+describe('getMatchingValue', () => {
+  const options = [
+    { label: 'Label1', value: 'value1' },
+    { label: 'Label2', value: 'value2' },
+    { label: 'Label3', value: 'value3' },
+  ]
+
+  it('should return the same value if it is not a string', () => {
+    const value = 42
+    // @ts-ignore
```

Comment:
can we not do this? i would prefer to cast as unknown as any. instead of fully ignoring